app/javascript/dashboard/routes/dashboard/captain/responses/Index.vue

```diff
@@ -292,34 +302,47 @@ onMounted(() => {
     <template #controls>
       <div
         v-if="shouldShowDropdown"
-        class="mb-4 -mt-3 flex justify-between items-center w-fit py-1"
+        class="mb-4 -mt-3 flex justify-between items-center w-full py-1"
```

Comment:
@scmmishra There seems to be a style issue with this change, could you check it out?
**Before**
<img width="553" height="149" alt="image" src="https://github.com/user-attachments/assets/72d459a2-3a0a-4868-be86-6d8b76dcb69e" />

**After**
<img width="1000" height="156" alt="image" src="https://github.com/user-attachments/assets/83f87e64-7b98-451a-a2ae-f5ad95058637" />